crates/ignore/src/default_types.rs

```diff
@@ -251,7 +251,7 @@ pub const DEFAULT_TYPES: &[(&str, &[&str])] = &[
     ("tex", &["*.tex", "*.ltx", "*.cls", "*.sty", "*.bib", "*.dtx", "*.ins"]),
     ("texinfo", &["*.texi"]),
     ("textile", &["*.textile"]),
-    ("tf", &["*.tf"]),
+    ("tf", &["*.tf", "*.auto.tfvars", "terraform.tfvars", "*.tf.json", "*.auto.tfvars.json", "terraform.tfvars.json", "*.terraformrc", "terraform.rc", "*.tfrc", "*.terraform.lock.hcl"]),
```

Comment:
Could you please follow the style of the surrounding code and make sure lines are wrapped to 79 columns inclusive?

Reply:
Hi, should be fine now, I think.

_& thanks for the quick response :smiley:, really like the tool_